pipeline.py

```diff
@@ -1,13 +1,26 @@
 """
 Pipeline for text processing implementation
 """
-
+import requests
 
 class EmptyDirectoryError(Exception):
     """
     No data to process
     """
 
+    def main():
```

Comment:
carefully read the docs. first part of assignment should be made within `scrapper.py` module. therefore, discard changes in `pipeline.py`